packages/flutter/lib/src/cupertino/text_selection_toolbar.dart

```diff
@@ -903,6 +940,11 @@ class _RenderCupertinoTextSelectionToolbarItems extends RenderBox with Container
         // already been taken care of when laying out the children to
         // accommodate the back button.
       }
+
+      // Update the controller values so that we can check in the horizontal
```

Comment:
I think the [Flutter style guide](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo) recommends not using "we" etc., so maybe say "...so that it's possible to check them in the horizontal..."

Reply:
Reminder on this one.